cyclonedx_py/_internal/environment.py

```diff
@@ -144,6 +156,24 @@ def __call__(self, *,  # type:ignore[override]
         self.__add_components(bom, rc, path=path)
         return bom
 
+    @staticmethod
+    def __collect_license_evidence(dist: Distribution) -> Optional[ComponentEvidence]:
+        if not (dist_files := dist.files):
+            return None
+
+        if not (license_files := [f for f in dist_files if LICENSE_FILE_REGEX.match(f.name)]):
```

Comment:
No need to guess the name of the file. 
You might be able to access all names via `dist.metadata.get_all('License-File')`
this is similar to how declared component licenses are collected.

Reply:
i see.
 then maybe do your auto-detection, and add the declared license files, too.

Reply:
Ok, thanks - so if I understand correctly we should try make a set out of the collected and maintainer-declared files right?

Reply:
you might consider preventing duplicates in certain means, yes.
To what extent? I'd leave the answer to you, as you are probably a user of this feature. I am certain you will make the right decision.